doc/api/util.md

```diff
@@ -300,11 +302,11 @@ Returns `true` if the given `object` is an `Array`. Otherwise, returns `false`.
 ```js
 const util = require('util');
 
-util.isArray([])
+util.isArray([]);
   // true
-util.isArray(new Array)
+util.isArray(new Array);
```

Comment:
when were the semicolons removed? the docs wg style guide says we should normally have them.